mautic/mautic
# Fix reset form field alias

| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | staging for features or enhancements / 3.2 for bug fixes 
| Bug fix?                               | yes/no
| New feature?                           | yes/no
| Deprecations?                          | yes/no
| BC breaks?                             | yes/no
| Automated tests included?              | yes/no
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 




#### Description:
Noticed issue  reset custom alias of field was reset every time while we edit it



#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Create new form
3. Create new field **Custom** with Field HTML name **custom1**
4. Save it
5. Open it and see Field HTML name is still **custom1**
6. Save again and see Field HTML name is now **custom**
7. We expect still **custom1** alias of field

![image](https://user-images.githubusercontent.com/462477/100260554-d81f9a80-2f49-11eb-8182-59e2b4a368ca.png)
